src/litdata/streaming/config.py

```diff
@@ -237,8 +237,9 @@ def __getitem__(self, index: ChunkedIndex) -> Tuple[str, int, int]:
 
         filesize_bytes = chunk["chunk_bytes"]
 
-        if self._config and self._config.get("encryption") is None and (not local_chunkpath.endswith(".parquet")):
-            filesize_bytes += (1 + chunk["chunk_size"]) * 4
+        # TODO: Check why filesize_bytes is calculated this way.
+        # if self._config and self._config.get("encryption") is None and (not local_chunkpath.endswith(".parquet")):
+        #     filesize_bytes += (1 + chunk["chunk_size"]) * 4
```

Comment:
I still have abit of doubt if this could have any backward compatibilty issue. 
Will do some more tests on this.

https://github.com/Lightning-AI/litdata/blob/012baa28d7707fefa0bc47c1cb8ecfc054fd352f/src/litdata/streaming/item_loader.py#L161-L164

Reply:
After debugging and analysis, it appears that there won't be any issues. Previously, `filesize_bytes` was smaller than the actual file size, but after the update, it will match the actual size. Therefore, the condition `os.stat(chunk_filepath).st_size >= filesize_bytes` would always be true  for the particular chunk file.